ldk/javascript/src/network/index.ts

```diff
@@ -29,6 +28,62 @@ export interface HTTPResponse {
   headers: Record<string, string[]>;
 }
 
+/**
+ * a simplified representation of a callback which take error
```

Comment:
For all of these comments above your interface, I think we should use sentence case ("This is sentence case")

Reply:
Updated